UnityProject/Assets/Scripts/Core/Chat/ChatEntry.cs

```diff
@@ -6,273 +6,246 @@
-		{
-			messageContentFitter.enabled = true;
-			LayoutRebuilder.ForceRebuildLayoutImmediate(entryTransform); // Poke fitter to set size on the same frame
-			yield return WaitFor.EndOfFrame; // But, for very large messages this isn't enough, so wait a frame...
-			messageContentFitter.enabled = false;
-		}
-
-		private IEnumerator WaitToCheckPos()
-		{
-			yield return WaitFor.EndOfFrame;
-
-			if (isHidden == false)
-			{
-				// Check to see if the chat entry is inside the viewport, and if so we will enable viewing it.
-				var isInsideViewport = entryTransform.rect.yMax.IsBetween(ViewportTransform.rect.yMin, ViewportTransform.rect.yMax);
-				isInsideViewport |= entryTransform.rect.yMin.IsBetween(ViewportTransform.rect.yMin, ViewportTransform.rect.yMax);
-				ToggleUIElements(isInsideViewport);
-			}
-
-			waitToCheck = null;
-		}
-
-		private IEnumerator FadeCooldown()
-		{
-			yield return WaitFor.Seconds(12f);
-			bool toggleVisibleState = false;
-
-			// Chat may have become focused during this time. Don't fade away if now focused.
-			if (IsChatFocused) yield break;
-
-			AnimateFade(0.01f, 3f);
-			if (isHidden == false)
-			{
-				SetHidden(true, true);
-				toggleVisibleState = true;
-			}
-
-			yield return WaitFor.Seconds(3f);
-
-			if (toggleVisibleState)
-			{
-				ToggleUIElements(false);
-			}
-		}
-
-		private IEnumerator AnimateStackObject()
-		{
-			yield return WaitFor.EndOfFrame;
-			var scaleFactor = Mathf.Min(0.1f * (stackCount - 1), 2);
-			var targetScale = new Vector3(stackScaleCache.x + scaleFactor, stackScaleCache.y + scaleFactor, 1);
-			var progress = 0f;
-			while (progress < 1f)
-			{
-				progress += Time.deltaTime * 10f;
-				stackObject.transform.localScale = Vector3.Lerp(stackScaleCache, targetScale, progress);
-				yield return WaitFor.EndOfFrame;
-			}
-
-			progress = 0f;
-			while (progress < 1f)
-			{
-				progress += Time.deltaTime * 10f;
-				stackObject.transform.localScale = Vector3.Lerp(targetScale, stackScaleCache, progress);
-				yield return WaitFor.EndOfFrame;
-			}
-
-			stackObject.transform.localScale = stackScaleCache;
-		}
-
-		private void SetStackPos()
-		{
-			if (messageText.textInfo.characterCount - 1 < messageText.textInfo.characterInfo.Length)
-			{
-				var lastCharacter = messageText.textInfo.characterInfo[messageText.textInfo.characterCount - 1];
-				var charWorld = messageText.transform.TransformPoint(lastCharacter.bottomRight);
-				var newWorldPos = stackObject.transform.position;
-				newWorldPos.x = charWorld.x + 3;
-				stackObject.transform.position = newWorldPos;
-			}
-			else
-			{
-				Logger.LogError("Caught an IndexOutOfRange in ChatEntry.SetStackPos() ln 273", Category.Chat);
-			}
-		}
-
-		private void AnimateFade(float toAlpha, float time)
-		{
-			messageText.CrossFadeAlpha(toAlpha, time, false);
-			stackText.CrossFadeAlpha(toAlpha, time, false);
-			stackImage.CrossFadeAlpha(toAlpha, time, false);
-		}
-	}
+namespace UI.Chat_UI {
```

Comment:
Same line doesn't really look nice and makes stuff and commmit too Compact

Reply:
Yeah this is awful